src/libraries/System.IO.Packaging/src/System/IO/Packaging/InternalRelationshipCollection.cs

```diff
@@ -320,7 +320,11 @@ private void ProcessRelationshipAttributes(XmlCompatibilityReader reader)
             {
                 try
                 {
+#if NET
```

Comment:
I had to lookup what `NET` means. I think it would be more self-describing to use `NET5_0_OR_GREATER`

Reply:
Done. The methods were actually added in 6 but the libraries did not target 5 which is going out of support soon.